ci-operator/step-registry/openshift-qe/installer/gcp/ipi/ovn/ipsec/openshift-qe-installer-gcp-ipi-ovn-ipsec-workflow.yaml

```diff
@@ -0,0 +1,13 @@
+workflow:
+  as: openshift-qe-installer-gcp-ipi-ovn-ipsec
+  steps:
+    post:
+    - chain: ipi-deprovision
+    pre:
+    - chain: ipi-conf-gcp
+    - ref: ovn-conf 
+    - ref: ovn-conf-ipsec-manifest
```

Comment:
I think ipsec and private are 2 different things: 

example flexy templates: 
https://gitlab.cee.redhat.com/aosqe/ci-profiles/-/blob/master/scale-ci/4.15/IPI-on-GCP-OVN-IPSEC.install.yaml?ref_type=heads
https://gitlab.cee.redhat.com/aosqe/ci-profiles/-/blob/master/scale-ci/4.15/IPI-on-GCP-Private.install.yaml?ref_type=heads

Reply:
Yes, you are right - ipsec and private are 2 different things :)